capstone/src/test/java/com/google/univiz/scorecard/CollegeDataApiImplTest.java

```diff
@@ -0,0 +1,56 @@
+package com.google.univiz.scorecard;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.inject.Guice;
+import com.google.inject.testing.fieldbinder.Bind;
+import com.google.inject.testing.fieldbinder.BoundFieldModule;
+import com.google.univiz.CollegeData;
+import com.google.univiz.api.CollegeId;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+import javax.inject.Inject;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public final class CollegeDataApiImplTest {
+
+  @Rule public MockitoRule rule = MockitoJUnit.rule();
+  @Bind @Mock CollegeIdReaderProvider mockReaderProvider;
+  @Bind ScorecardConverter scorecardConverter = new ScorecardConverter();
```

Comment:
I don't think you'll need this. Guice will be able to inject it by default.

Reply:
After experimentation, this didn't work.

Reply:
Am I missing something in the `setup` function?

Reply:
This works for me when I tried it locally. What's the error that you're observing?

Reply:
```
1) Binding to null values is only allowed for fields that are annotated @Nullable.
  at com.google.univiz.scorecard.CollegeDataApiImplTest.scorecardConverter(CollegeDataApiImplTest.java:27)
```